src/gui/editors/TimeLineWidget.cpp

```diff
@@ -88,9 +88,10 @@ void TimeLineWidget::setXOffset(const int x)
 void TimeLineWidget::addToolButtons( QToolBar * _tool_bar )
 {
 	auto autoScroll = new NStateButton(_tool_bar);
-	autoScroll->setGeneralToolTip( tr( "Auto scrolling" ) );
-	autoScroll->addState( embed::getIconPixmap( "autoscroll_on" ) );
-	autoScroll->addState( embed::getIconPixmap( "autoscroll_off" ) );
+	autoScroll->setGeneralToolTip(tr("Auto scrolling"));
+	autoScroll->addState(embed::getIconPixmap("autoscroll_stepped_on"), "Stepped auto scrolling");
+	autoScroll->addState(embed::getIconPixmap("autoscroll_continuous_on"), "Continuous auto scrolling");
+	autoScroll->addState(embed::getIconPixmap("autoscroll_off"), "Auto scrolling disabled");
```

Comment:
These tooltips should be translatable:
```suggestion
	autoScroll->addState(embed::getIconPixmap("autoscroll_stepped_on"), tr("Stepped auto scrolling"));
	autoScroll->addState(embed::getIconPixmap("autoscroll_continuous_on"), tr("Continuous auto scrolling"));
	autoScroll->addState(embed::getIconPixmap("autoscroll_off"), tr("Auto scrolling disabled"));
```